packages/@webex/internal-plugin-voicea/test/unit/spec/voicea.js

```diff
@@ -67,11 +66,20 @@ describe('plugin-voicea', () => {
             eventType: 'relay.event',
             relayType: 'client.annc',
           },
-          trackingId: sinon.match.string
+          trackingId: sinon.match.string,
         });
       });
     });
 
+    describe('#deregisterEvents', () => {
+      it('deregisters voicea service', () => {
+        voiceaService.deregisterEvents();
+        assert.equal(voiceaService.hasVoiceaJoined, false);
```

Comment:
are these values set to anything different before the call to `voiceaService.deregisterEvents();` is done? I don't see that

Reply:
my bad, added now